src/test/java/io/jenkins/plugins/artifact_manager_jclouds/MockApiMetadata.java

```diff
@@ -193,12 +190,13 @@ public boolean blobExists(String container, String key) {
         }
 
         @Override
-        public Iterable<String> getBlobKeysInsideContainer(String container) throws IOException {
+        public Iterable<String> getBlobKeysInsideContainer(String container, String prefix) throws IOException {
```

Comment:
adapts to https://github.com/apache/jclouds/commit/29eec441e902162394a5dbceaf98c14d3b2bc87a